src/components/parts/frontpage-shortcuts/FrontpageShortcuts.tsx

```diff
@@ -12,29 +12,58 @@ import saksbehandlingstider from '/public/gfx/front-page-shortcuts/saksbehandlin
 import utbetalingsdatoer from '/public/gfx/front-page-shortcuts/utbetalingsdatoer_nav_ikon.svg';
 import pengestotter from '/public/gfx/front-page-shortcuts/pengestotter_og_tjenester_fra_a_til_a_nav_ikon.svg';
 import soknader from '/public/gfx/front-page-shortcuts/soknader_og_skjema_nav_ikon.svg';
+import arbeidsgiverMinside from '/public/gfx/front-page-shortcuts/arbeidsgiver_minside.svg';
+import arbeidsgiverSoknader from '/public/gfx/front-page-shortcuts/arbeidsgiver_soknader.svg';
+import arbeidsgiverTjenester from '/public/gfx/front-page-shortcuts/arbeidsgiver_tjenester.svg';
+import { ContentProps, ContentType } from 'types/content-props/_content-common';
 
 import style from './FrontpageShortcuts.module.scss';
 
-export const FrontpageShortcuts = ({ config }: FrontpageShortcutsProps) => {
+const linkToIconDictionary = {
+    person: {
+        saksbehandlingstider: saksbehandlingstider,
+        utbetalingsdatoer: utbetalingsdatoer,
+        tjenester: pengestotter,
+        'soknader/nb/person': soknader,
+    },
+    employer: {
+        'min-side-arbeidsgiver': arbeidsgiverMinside,
+        'soknader/nb/bedrift': arbeidsgiverSoknader,
+        tilganger: arbeidsgiverTjenester,
+    },
+};
```

Comment:
Burde helst ikke hardkode disse.

Reply:
Ja, vet. Det er veldig skjørt. Men Bjørn har en oppgave på det, så fremfor å la scope vokse for mye så gjorde jeg det på samme måte for nå og så høre hva han har tenkt...